ntc_templates/templates/index

```diff
@@ -716,6 +716,7 @@ huawei_smartax_display_ont_port_state_0_1_eth-port.textfsm, .*, huawei_smartax,
 huawei_smartax_display_ont_port_vlan_0_1_byvlan_0.textfsm, .*, huawei_smartax, di[[splay]] ont p[[ort]] v[[lan]] \d+ \d+ byvlan \d+
 huawei_smartax_display_ont-lineprofile_gpon_all.textfsm, .*, huawei_smartax, disp[[lay]] ont-li[[neprofile]] g[[pon]] a[[ll]]\s*
 huawei_smartax_display_mac-address_ont_0_1_2_0.textfsm, .*, huawei_smartax, di[[splay]] mac-a[[ddress]] o[[nt]] \d+(\/)?\s*\d+(\/)?\s*\d+ \d+ *$
+huawei_smartax_display_ont-srvprofile_gpon_all.textfsm, .*, huawei_smartax, disp[[lay]] ont-s[[rvprofile]] g[[pon]] a[[ll]]\s*
```

Comment:
If one smartax command can have the `display` word shortened to `di`, then I'd expect that holds true for all of them, right?

```suggestion
huawei_smartax_display_ont-lineprofile_gpon_all.textfsm, .*, huawei_smartax, di[[splay]] ont-li[[neprofile]] g[[pon]] a[[ll]]\s*
huawei_smartax_display_mac-address_ont_0_1_2_0.textfsm, .*, huawei_smartax, di[[splay]] mac-a[[ddress]] o[[nt]] \d+(\/)?\s*\d+(\/)?\s*\d+ \d+ *$
huawei_smartax_display_ont-srvprofile_gpon_all.textfsm, .*, huawei_smartax, di[[splay]] ont-s[[rvprofile]] g[[pon]] a[[ll]]\s*
```